### ISSUE TITLE
Error after working for some time

### ISSUE THREAD

**Issue by chinciusan** [social-manager-tools/instagram-bot.js#51]
### Versions

Replace the values below with your own:

- **Bot Version:** v0.7.1 and 0.7.3
Raspberry Pi

### Expected Behavior

Please describe the program's expected behavior.

```

```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
[INFO] like_realistic: time night: 1648
(node:17340) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'replace' of undefined
    at Likemode_realistic.start (/home/pi/Projects/newbots/ralea2/modules/mode/likemode_realistic.js:155:143)
    at switch_mode (/home/pi/Projects/newbots/ralea2/bot.js:85:48)
    at start (/home/pi/Projects/newbots/ralea2/bot.js:114:19)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
(node:17340) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
(node:17340) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```
Everything was working fine for a while. Maybe Instagram changed something?

**Comment by ptkdev** [comment]
This happen when hashtag don't have photo or profile is locked. I miss this check in code.

I fix this in **0.7.5** but at moment [try this code](https://pastebin.com/raw/VRk1Xv88): replace it in file `modules/mode/likemode_realistic.js`

**Comment by ptkdev** [comment]
P.s. @chinciusan if you update from **0.7.1** -> **0.7.3** you need add new line of code to `config.js` from `config.js.tpl`. You debug code have "replace error" of missing `bot_start_sleep` variable from config.

**Comment by ptkdev** [comment]
**TODO**: 0.7.4 default value of config.js if don't exist

**Comment by ptkdev** [comment]
Fixed in **[0.7.4 BETA](https://github.com/social-manager-tools/instagram-bot.js/archive/master.zip)**

<a href="http://patreon.ptkdev.io" alt="Patreon Backer"><img src="https://img.shields.io/badge/donate-patreon-F87668.svg"></a> <a href="http://paypal.ptkdev.io" alt="Paypale Donate"><img src="https://img.shields.io/badge/donate-paypal-46AFE0.svg"></a>
